Tensile/TensileCreateLibrary.py

```diff
@@ -122,12 +122,25 @@ def buildSourceCodeObjectFile(outputPath, kernelFile):
 
     archFlags = ['-amdgpu-target=' + arch for arch in archs]
 
-    hipFlags = subprocess.check_output([which('hcc-config'), '--cxxflags', '--shared']).decode().split(' ')
-    hipLinkFlags = subprocess.check_output([which('hcc-config'), '--ldflags', '--shared']).decode().split(' ')
+    if (CxxCompiler == 'hcc'):
+
+      hipFlags = subprocess.check_output([which('hcc-config'), '--cxxflags', '--shared']).decode().split(' ')
+      hipLinkFlags = subprocess.check_output([which('hcc-config'), '--ldflags', '--shared']).decode().split(' ')
+
+      hipFlags += ['-I', outputPath]
+
+      compileArgs = [which('hcc')] + hipFlags + archFlags + [kernelFile, '-c', '-o', objectFilepath]
+
+    elif (CxxCompiler == "hipcc"):
+
+      hipFlags = "--genco"
+      hipLinkFlags = ""
+
+      hipFlags += ['-I', outputPath]
+
+      compileArgs = [which('hipcc')] + hipFlags + archFlags + [kernelFile, '-c', '-o', objectFilepath]
```

Comment:
hip-clang does not need subprocess.check_call(linkArgs) and subprocess.check_call(extractArgs, cwd=buildPath).
the output of compileArgs should be soFilename instead of objectFilepath